lib/prism.css

```diff
@@ -98,70 +98,70 @@ pre[class*="language-"] {
 .language-css .token.string,
 .style .token.string,
 .token.variable {
-	color: #a67f59;
-	background: hsla(0,0%,100%,.5);
+        color: #a67f59;
+        background: hsla(0,0%,100%,.5);
 }
 
 .token.atrule,
 .token.attr-value,
 .token.keyword {
-	color: #07a;
+        color: #07a;
 }
 
 
 .token.regex,
 .token.important {
-	color: #e90;
+        color: #e90;
 }
 
 .token.important {
-	font-weight: bold;
+        font-weight: bold;
 }
 
 .token.entity {
-	cursor: help;
+        cursor: help;
 }
 
 pre.line-numbers {
-	position: relative;
-	padding-left: 3.8em;
-	counter-reset: linenumber;
+        position: relative;
+        padding-left: 3.8em;
+        counter-reset: linenumber;
 }
 
 pre.line-numbers > code {
-	position: relative;
+        position: relative;
 }
 
 .line-numbers .line-numbers-rows {
-	position: absolute;
-	pointer-events: none;
-	top: 0;
-	font-size: 100%;
-	left: -3.8em;
-	width: 3em; /* works for line-numbers below 1000 lines */
-	letter-spacing: -1px;
-	border-right: 1px solid #999;
-
-	-webkit-user-select: none;
-	-moz-user-select: none;
-	-ms-user-select: none;
-	user-select: none;
-
-}
-
-	.line-numbers-rows > span {
-		pointer-events: none;
-		display: block;
-		counter-increment: linenumber;
-	}
-
-		.line-numbers-rows > span:before {
-			content: counter(linenumber);
-			color: #999;
-			display: block;
-			padding-right: 0.8em;
-			text-align: right;
-		}
+        position: absolute;
+        pointer-events: none;
+        top: 0;
+        font-size: 100%;
+        left: -3.8em;
+        width: 3em; /* works for line-numbers below 1000 lines */
+        letter-spacing: -1px;
+        border-right: 1px solid #999;
+
+        -webkit-user-select: none;
+        -moz-user-select: none;
+        -ms-user-select: none;
+        user-select: none;
+
+}
+
+.line-numbers-rows > span {
+        pointer-events: none;
+        display: block;
+        counter-increment: linenumber;
+}
+
+.line-numbers-rows > span:before {
+        content: counter(linenumber);
+        color: #999;
+        display: block;
+        padding-right: 0.8em;
+        text-align: right;
+}
 .token a {
-	color: inherit;
+        color: inherit;
 }
```

Comment:
There  have been some useless conflicting errors. I has initially filed a PR #367 which had already deleted these prism.js and prism.css files. So merging them wont be a problem .

Reply:
Can you get update your fork from the current master and get rid of the extra changes here?